lib/rules/inline-link-to.js

```diff
@@ -1,20 +1,42 @@
 'use strict';
 
+const { builders: b } = require('ember-template-recast');
+
 const Rule = require('./base');
 
 const message = 'The inline form of link-to is not allowed. Use the block form instead.';
 
 module.exports = class InlineLinkTo extends Rule {
   visitor() {
     return {
-      MustacheStatement(node) {
+      MustacheStatement(node, { parentNode, parentKey }) {
         if (node.path.original === 'link-to') {
-          this.log({
-            message,
-            line: node.loc && node.loc.start.line,
-            column: node.loc && node.loc.start.column,
-            source: this.sourceForNode(node),
-          });
+          if (this.mode === 'fix') {
+            let titleNode = node.params[0];
+
+            let newBody;
+            if (titleNode.type === 'SubExpression') {
+              newBody = b.mustache(titleNode.path, titleNode.params, titleNode.hash);
+            } else if (titleNode.type === 'StringLiteral') {
+              newBody = b.text(titleNode.value);
+            }
```

Comment:
What about other types (e.g. `{{link-to 1234 'blah.route'}}`), should we add an `else` and log the error?

Reply:
how do we log the error in this mode? the regular `this.log()` again?

Reply:
done, I think 😅